packages/expo-video/src/VideoPlayer.types.ts

```diff
@@ -40,6 +40,30 @@ export declare class VideoPlayer extends SharedObject<VideoPlayerEvents> {
    */
   currentTime: number;
 
+  /**
+   * The current time of the video as a date. The system calculates this value from the `EXT-X-PROGRAM-DATE-TIME` tag.
+   * If a livestream doesn't have the required metadata, this will return `null`.
+   * > This property is get-only
+   * @platform android
+   * @platform ios
+   */
+  readonly currentDate: number | null;
```

Comment:
nitpick/question: Shouldn't this be a `Date` type or a differently named property? 
`currentUNIXTimestamp` `currentTimestamp` come to mind but not sure these are better.

@behenate wdyt?

Reply:
It shouldn't be the `Date` type since it's not a JS date object, but a number. So the user has to manually create a `Date` object using `new Date()`.

We can potentially rename it though.

Reply:
I mean, should we return it from the API as a Date?

Reply:
I guess since it's called `currentDate` it would be better to return a date from it. I kind of like `currentTimestamp` better though @tsapeta what do you think? 😅

Reply:
If you all agree I'll rename it to `currentTimestamp` 🙂